src/compositemark/boxplot.ts

```diff
@@ -186,7 +186,7 @@ export function normalizeBoxPlot(
   }
   return {
     ...outerSpec,
-    transform,
+    transform: [...(outerSpec.transform || []), ...transform],
```

Comment:
Hmm, maybe it's cleaner to write `(outerSpec.transform || []).concat(transform)` and use concat below as well.

Reply:
What do you mean below?

Reply:
The other line you changed

Reply:
transform in error bar has more than 1 elements to be combined. Would it be better to use this instead of `concat` in errorbar.ts?

Reply:
You're right. It's good as it is.